lib/requester.js

```diff
@@ -19,7 +19,10 @@ exports.do = (input, accessToken, apiUrl) => {
       }
       if (res.statusCode < 200 || res.statusCode > 299) {
         const errMsg = `Failed to get response, status code ${res.statusCode}`
-        if (!data || !data.details) {
+        if (res.statusCode === 413) {
+          reject(new Error('The requsted input data is too large to process.'))
```

Comment:
Is it input data, or is it the errors that resulted from the input data?

Reply:
@rocky the error message should be taken from the `error` field of response body.

Reply:
Just looked at https://staging.api.mythx.io/v1/openapi#operation/submitAnalysis 

Sigh

> "Payload Too Large"

This is something that is meaningful to an end user? I guess it doesn't have a spelling mistake like "reqsted" and is not _incorrect_ about that this is coming from the server side as opposed to the user side, but it is vague. If I were in the business of giving helpful useful messages (well, I guess I am) this is not what I would offer. 

Correct me if I am wrong, but the particular situation here is that there are so many security warnings and messages generated in the analysis, that it is too large to fit in the response.  So if that is the case, why can't we say something closer to this? 

Useful and informative messages gives the user some sense as to what the problem is or how likely it is to correct.  As a result of understanding the situation, the smart contract programmer is empowered to make decisions such as to switch from full analysis to quick analysis, or cut down my contract to a smaller pieces. 

@birdofpreyru I saw that had closed BAH-66 without having it reviewed by anyone, and I suspect that also is true of the underlying git commit. This is not great practice.  To avoid the vagueness problems I've seen again and again in the openapi doc. (the description of clientToolName is another example) why not have someone review the descriptions in advance?  

BTW Where is the PR that is assocated with this?  That wold be a good place to start for someone reviewing descriptions.

Reply:
No, 413 is only too large input. If the response body is too large, it would be some 5xx error. Most probably 504.

Reply:
Ok. So are you saying that the specific situation the JSON that sent was too big? 

If so, then let's say that.  Let' s not than leave vague which direction we are talking about or leave vague what "Payload" means. Here we are talking about JSON smart contract data. 

The suggested action is still almost that same - reduce the size of the input by sending fewer contracts or breaking the contracts down into smaller pieces for analysis. 

(Changing analysis mode isn't going to do squat if this is an input problem)

Reply:
@birdofpreyru when the API responses are impropved to be more helpful and specific we will come back to this and do as you say and use what is in the API's `error` field. Until then we will workaround this by setting here.